praetorian_cli/handlers/run.py

```
@@ -175,35 +175,119 @@ def tool(sdk, tool_name, target, extra_config, credential, wait, use_agent, loca
-    click.echo(f'{"─"*16} {"─"*50}')
-    for name, desc in sorted(FRIENDLY_NAMES.items()):
-        click.echo(f'{name:<16} {desc}')
-    click.echo(f'\nUse "guard run capabilities" for the full list of backend capabilities.')
+    from praetorian_cli.runners.local import INSTALLABLE_TOOLS, list_installed
+
+    try:
+        result = sdk.capabilities.list()
+        caps = result[0] if isinstance(result, tuple) else result
+        if not isinstance(caps, list):
+            caps = []
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)